### PR TITLE
[12.0][MIG] sale_order_for_approval

### PR DESCRIPTION
**Pull request by carmenbianca**
Task: https://gestion.coopiteasy.be/web#id=7882&view_type=form&model=project.task&action=475&active_id=210

Not sure how to find report_saleorder_document through the GUI; will have to test later.

### CODE REVIEW

**Comment by carmenbianca** [comment]
Tested; works.

File: sale_order_for_approval/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# © 2017- Robin Keunen - Coop IT Easy SCRLfs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Order - For Approval Mention",
+    "version": "12.0.1.0.0",
+    "category": "Sales",
+    "sequence": 95,
```
Comment (huguesdk) [inline sale_order_for_approval/__manifest__.py]: i don’t think this is useful anymore, but i might be mistaken.

File: sale_order_for_approval/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "author": "Coop IT Easy SCRLfs",
+    "website": "https://coopiteasy.be",
+    "summary": 'Display "For Approval" mention on Sale Orders',
+    "depends": ["sale"],
+    "data": [
+        "views/report_saleorder.xml",
+    ],
+    "installable": True,
```
Comment (huguesdk) [inline sale_order_for_approval/__manifest__.py]: true by default, should be removed.

**Review by huguesdk — CHANGES_REQUESTED** [review]
except those slight changes is the manifest, lgtm. 👍

**Review by robinkeunen — APPROVED** [review]

**Comment by robinkeunen** [comment]
To find `report_saleorder_document` in the UI, you need to find the menuitem calling to the report action and follow its parents hierarchy.

**Pull request merged**